streams/src/test/java/org/apache/kafka/streams/integration/StoreQueryIntegrationTest.java

```diff
@@ -332,6 +332,7 @@ private Properties streamsConfiguration() {
         config.put(ConsumerConfig.HEARTBEAT_INTERVAL_MS_CONFIG, 200);
         config.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, 1000);
         config.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 100);
+        config.put(StreamsConfig.NUM_STREAM_THREADS_CONFIG, 2);
```

Comment:
We should write a proper test case instead of "piggy-backing" it into an existing test.

Reply:
Added dedicated test